orderer/consensus/etcdraft/blockpuller.go

```diff
@@ -107,3 +119,108 @@ func NewBlockPuller(support consensus.ConsenterSupport,
 		BlockPuller:    bp,
 	}, nil
 }
+
+// NewBlockFetcher creates a new block fetcher
+func NewBlockFetcher(support consensus.ConsenterSupport,
+	baseDialer *cluster.PredicateDialer,
+	clusterConfig localconfig.Cluster,
+	bccsp bccsp.BCCSP,
+) (BlockPuller, error) {
+	verifyBlockSequence := func(blocks []*common.Block, _ string) error {
+		return cluster.VerifyBlocks(blocks, support)
```

Comment:
we need to update the support as we pull blocks. We cannot re-use the support from the constructor.